changelog/1.37.0.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "1.37.0"
+description: "Released on 11/23/2022"
+---
+
+### Breaking changes ❗
+
+- infra: Workspace statistics were previously stored in a table
+  `environment_stats`. This table is removed in this release. Downgrading from
+  this version to an older version may require manual steps.
+- cli: `coder ssh` and `coder tunnel` invocations from older versions of the CLI
+  may fail with the following error:
+
+  ```console
+  cannot unmarshal string into Go struct field ICEServer.data.credentialType of
+  type webrtc.ICECredentialType
+  ```
```

Comment:
is this still broken? I'm able to ssh just fine with 1.36.0 CLI to the RC

Reply:
Discussed: should be OK for this release as the breaking JSON marshalling change isn't in the version of `pion/webrtc` used in this release.